gitops.dockerfile

```diff
@@ -0,0 +1,31 @@
+# Go build
+FROM golang:1.17 AS go-build
+# Add a kubectl
+RUN apt-get install -y apt-transport-https ca-certificates curl openssh-client && \
+    curl -fsSLo /usr/share/keyrings/kubernetes-archive-keyring.gpg \
+    https://packages.cloud.google.com/apt/doc/apt-key.gpg && \
+    echo "deb [signed-by=/usr/share/keyrings/kubernetes-archive-keyring.gpg] \
+    https://apt.kubernetes.io/ kubernetes-xenial main" | tee /etc/apt/sources.list.d/kubernetes.list && \
+    apt-get update && \
+    apt-get install -y kubectl
+# Add known_hosts entries for GitHub and GitLab
+RUN mkdir ~/.ssh
+RUN ssh-keyscan github.com >> ~/.ssh/known_hosts
+RUN ssh-keyscan gitlab.com >> ~/.ssh/known_hosts
+
+COPY Makefile /app/
+COPY tools /app/tools
+WORKDIR /app
+RUN make dependencies
```

Comment:
Do we really need to add make dependencies here? This is the one that starts using sudo inside your container.

Reply:
fixed